v2/cmd/nuclei/main.go

```diff
@@ -238,7 +241,18 @@ on extensive configurability, massive extensibility and ease of use.`)
 	if options.LeaveDefaultPorts {
 		http.LeaveDefaultPorts = true
 	}
-
+	if options.CustomConfigDir != "" {
+		config.SetCustomConfigDirectory(options.CustomConfigDir)
+		configPath := path.Join(options.CustomConfigDir, "config.yaml")
+	readConfigFile:
```

Comment:
I think it's better to call the required function twice in case of errors since if, for some reason, the config file can't be copied, the code will be stuck in a loop

Reply:
Done